training/distributed_training/pytorch/model_parallel/gpt2/smp-train-gpt-simple.ipynb

```diff
@@ -0,0 +1,716 @@
+{
```

Comment:
<p>Nit - two mentions of "for CI" - an external user will not know what that means. Elaborate a bit to something like "for continuous integration testing"?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3034/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
<p>See my comment in the previous section.</p>